bin/mail/2016-09-27-latam-hoc-minecraft/generate-mailing-lists.rb

```diff
@@ -0,0 +1 @@
+../2016-09-15-latam-hoc-starwars/generate-mailing-lists.rb
```

Comment:
This will be a new .csv file. There is no `latam-contacts.csv` to replace, because:
 1. This mailing is in a new directory,
 2. I always delete the generated .csv files once finished. They are not committed to the repo. We keep a record of the email recipients in the DB.